src/test/java/org/jitsi/meet/test/base/AbstractParticipantHelper.java

```diff
@@ -133,22 +201,39 @@ public void waitForSecondParticipantToConnect()
 
         if (participant2.isHungUp())
         {
-            participant2.joinConference(currentRoomName);
+            participant2.joinConference(currentRoomName, fragment);
         }
 
         MeetUtils.waitForParticipantToJoinMUC(participant2.getDriver(), 10);
-        MeetUtils.waitForIceConnected(participant2.getDriver());
-        MeetUtils.waitForSendReceiveData(participant2.getDriver());
-
-        TestUtils.waitMillis(5000);
     }
 
     /**
      * Starts the owner, if it isn't started and hangups all other participants.
      */
-    public void closeAllParticipantsExceptTheOwner()
+    public void hangUpAllParticipantsExceptTheOwner()
+    {
+        waitForParticipant1ToJoinMUC(null, null);
+
+        if (participant2 != null && !participant2.isHungUp())
```

Comment:
it asks for "hangupParticipant2" method, see how this logic is duplicated in many other places...